virttest/qemu_vm.py

```diff
@@ -1533,10 +1552,11 @@ def sort_key(dev):
         if not amd_vendor_string:
             amd_vendor_string = "AuthenticAMD"
         if amd_vendor_string == cpu.get_cpu_vendor():
-            # AMD cpu do not support multi threads.
-            if params.get("test_negative_thread", "no") != "yes":
+            # AMD cpu do not support multi threads besides EPYC
+            if (params.get("test_negative_thread", "no") != "yes" and
+                    not cpu_model.startswith('EPYC')):
```

Comment:
`cpu_model` maybe `None` on other architectures, so you would better to check at first or use `(cpu_model or '').startswith` to handle this problem.

Reply:
@PaulYuuu Thanks, I have noticed that so updated the definition of `cpu_model` to `cpu_model = params.get("cpu_model", "")` at Line 1513, that should okay.

Reply:
Okay, I see. Then this line also needs to be modified: `cpu_model = params.get("default_cpu_model")`

Reply:
@PaulYuuu Yes you are right, modifying. Thanks.